--- nagios-vshell-1.8/config/vshell.conf~	2011-08-12 00:46:53.000000000 +0300
+++ nagios-vshell-1.8/config/vshell.conf	2012-01-07 18:53:03.159606028 +0200
@@ -18,16 +18,16 @@
 COREURL = "nagios"
 
 ; Full filesystem path to the Nagios status file
-STATUSFILE = "/usr/local/nagios/var/status.dat"
+STATUSFILE = "/var/lib/nagios/status.dat"
 
 ; Full filesystem path to the Nagios object cache file
-OBJECTSFILE = "/usr/local/nagios/var/objects.cache"
+OBJECTSFILE = "/var/lib/nagios/objects.cache"
 
 ; Full filesystem path to the Nagios CGI permissions configuration file
-CGICFG = "/usr/local/nagios/etc/cgi.cfg"
+CGICFG = "/etc/webapps/nagios/cgi.cfg"
 
 ; Full filesystem path to the Nagios command pipe
-NAGCMD = "/usr/local/nagios/var/rw/nagios.cmd"
+NAGCMD = "/var/lib/nagios/rw/nagios.cmd"
 
 ; Full filesystem path to the Nagios lock file (when Nagios is running)
 NAGLOCK = "/usr/local/nagios/var/nagios.lock"
--- nagios-vshell-1.8/constants.inc.php~	2011-09-19 19:52:27.000000000 +0300
+++ nagios-vshell-1.8/constants.inc.php	2012-01-07 18:58:30.123327515 +0200
@@ -66,7 +66,7 @@
 // If necessary, adjust the path to vshell.conf, but nothing else below that
 // All other setting should be adjusted by editing vshell.conf
 // Switch to use external configuration file by Tony Yarusso, 30 March 2011
-$ini_array = parse_ini_file("/etc/vshell.conf");
+$ini_array = parse_ini_file("/etc/webapps/nagios-vshell/vshell.conf");
 
 define('VERSION', '1.8'); 
 
